### ISSUE TITLE
Implement handling of board data after a board update request [4/15]

### ISSUE THREAD

**Issue by RobertDuras** [Enari/DVA313#113]
Create functionality for handling board data once a board update has been discovered.
This relates to reading the returned data from the API and having it be reflected on the board.
This feels like a different part of the kanban board functionality, so I'm splitting it into a separate issue.

Dependent on #55
